parts/kubernetesmastercustomscript.sh

```diff
@@ -103,32 +103,70 @@ function setDockerOpts () {
     sed -i "s#^DOCKER_OPTS=.*#DOCKER_OPTS=${1}#" /etc/default/kubelet
 }
 
-function configNetworkPolicy() {
+function configAzureNetworkPolicy() {
+    # Download netconfig.
+    NETCONFIGDIR=/etc/cni/net.d
+    NETCONFIGFILE=$NETCONFIGDIR/10-azure.conf
+    mkdir -p $NETCONFIGDIR
+
+    echo "{
+  \"cniVersion\": \"0.2.0\",
+  \"name\": \"azure\",
+  \"type\": \"azure-vnet\",
+  \"master\": \"eth0\",
+  \"bridge\": \"azure0\",
+  \"ipam\": {
+    \"type\": \"azure-vnet-ipam\"
+  }
+}
+" > $NETCONFIGFILE
+
+    chown -R root:root $NETCONFIGDIR
+    chmod 755 $NETCONFIGDIR
+    chmod 644 $NETCONFIGFILE
+
+    # Download Azure VNET CNI plugins.
+    CNIBINDIR=/opt/cni/bin
+    mkdir -p $CNIBINDIR
+    curl -fsSL --retry 12 --retry-delay 10 https://github.com/Azure/azure-container-networking/releases/download/v0.7/azure-cni-linux-amd64-v0.7.tgz | tar -xz -C $CNIBINDIR
+    curl -fsSL --retry 12 --retry-delay 10 https://github.com/containernetworking/cni/releases/download/v0.4.0/cni-amd64-v0.4.0.tgz | tar -xz -C $CNIBINDIR ./loopback
```

Comment:
Should we "externalize" these versions, at least to variables at the top of this script? `$CNI_VERSION` or something?